Wox.Core/Updater.cs

```diff
@@ -52,31 +55,47 @@ public async Task UpdateApp()
                 return;
             }
 
-            var fr = u.FutureReleaseEntry;
-            var cr = u.CurrentlyInstalledVersion;
-            Log.Info($"|Updater.UpdateApp|Future Release <{fr.Formatted()}>");
-            if (fr.Version > cr.Version)
+            var newReleaseVersion = Version.Parse(u.FutureReleaseEntry.Version.ToString());
```

Comment:
Any reason not to use the previous method to compare versions?

Reply:
u.FutureReleaseEntry.Version returns SemanticVersion (which we wont be using going forward), and we compare on Version property.

Also think when I was testing, debug mode didn't play well when Squirrel tries to retrieve the current release version. So getting it from Wox itself made more sense.

Reply:
will it break squirrel?

Reply:
Nah it won't break squirrel, it works with it